src/javascript/app_2/Stores/Modules/Trading/trade_store.js

```diff
@@ -120,6 +121,15 @@ export default class TradeStore extends BaseStore {
                 this.changeDurationValidationRules();
             },
         );
+        // Adds intercept to update query strings and session storage properties 'start_time with '00:00'
+        reaction(
```

Comment:
@kellybinary 
we can set `start_time` to `null` when `contract_start_type` is `spot` in `processNewValuesAsync` function instead of adding reaction.
If `start_time` is changed it in `processNewValuesAsync`, it'll prevent react from doing extra render when `contract_start_type` is changed from `forward` to `spot` because both of changes are done in one transaction. Also, we keep all the logic in one place.